.gitignore

```diff
@@ -69,6 +69,7 @@ build-*
 .mesonpy-native-file.ini
 # sphinx build directory
 _build
+_contents
```

Comment:
this seems unrelated

```suggestion
```

Reply:
Oh I missed that part.
Technically speaking, it would be cleanest to open a separate PR for that, but I personally wouldn't mind it if you kept it in here, given the size of the change and all.